pycode/memilio-epidata/memilio/epidata/getDataIntoPandasDataFrame.py

```diff
@@ -111,6 +110,7 @@ def __init__(self, out_folder, **kwargs):
             self.interactive = True if kwargs['interactive'] == 'True' else False
             self.plot = True if kwargs['make_plot'] == 'True' else False
             self.no_raw = True if kwargs['no_raw'] == 'True' else False
+            self.to_dataset = True if kwargs['to_dataset'] == 'True' else False
```

Comment:
```suggestion
            self.interactive = True if str(kwargs['interactive']) == 'True' else False
            self.plot = True if str(kwargs['make_plot']) == 'True' else False
            self.no_raw = True if str(kwargs['no_raw']) == 'True' else False
            self.to_dataset = True if str(kwargs['to_dataset']) == 'True' else False
```

The dtype of kwarg values differs, dependant on where they come from. I would suggest this, as it covers all possible cases. 
Line 107 & 109 for show_prog and self.checks should also work like this, I just cant make a suggestion there.